### PR TITLE
Feature : Add RTL languages support

### PR DESCRIPTION
**Pull request by ansarikhurshid786**
rtl automatically detect and show accordingly

### CODE REVIEW

**Comment by guhungry** [comment]
In iOS, it seems I need to add alignment in TextStyle like in android in [ios-photo-manipulator](https://github.com/guhungry/ios-photo-manipulator/blob/master/Sources/WCPhotoManipulator/Models/TextStyle.swift) and implement the functionality.
So, you can use a similar solution in Android instead of implementing low-level details here and missing other features like rotation, text shadow, etc.

**Comment by ansarikhurshid786** [comment]
@guhungry i was testing my changes it was working. can u check my changes and support rtl content

**Review by guhungry — APPROVED** [review]
Confirm worked in #943

File: android/src/main/java/com/guhungry/rnphotomanipulator/RNPhotoManipulatorModule.kt [hunk 1]
```diff
@@ -174,19 +174,49 @@ class RNPhotoManipulatorModule(private val context: ReactApplicationContext) : R
...
+            toColorInt(options.getMap("shadowColor"))         // Shadow color
         )
-        printText(image, text, location, style)
+
+        // Print the text on the image with the adjusted location and style
+        printText(image, bidiFormatter.unicodeWrap(text), adjustedLocation, style)
+    }
+
+    // Helper function to detect Arabic characters in a string
+    private fun containsArabic(text: String): Boolean {
```
[the remark this answers is not in the document]
Comment (guhungry) [reply android/src/main/java/com/guhungry/rnphotomanipulator/RNPhotoManipulatorModule.kt]: This method is removed

File: ios/RNPhotoManipulatorImpl.m [hunk 2]
```diff
@@ -187,15 +187,64 @@ + (void)printText:(NSString *)uri
...
+            image = UIGraphicsGetImageFromCurrentImageContext();
+            UIGraphicsEndImageContext();
       }
 
       NSString *uri = [ImageUtils saveTempFile:image mimeType:mimeType quality:DEFAULT_QUALITY];
       resolve(uri);
     }];
 }
 
++ (BOOL)isTextRTL:(NSString *)text {
```
[the remark this answers is not in the document]
Comment (guhungry) [reply ios/RNPhotoManipulatorImpl.m]: This method is removed too

**Pull request merged**